src/Raven.Client/Documents/Smuggler/DatabaseItemType.cs

```diff
@@ -11,6 +11,6 @@ public enum DatabaseItemType
         RevisionDocuments = 1 << 1,
         Indexes = 1 << 2,
         Transformers = 1 << 3,
-        Identities = 1 << 4,
+        ClusterIdentities = 1 << 4
```

Comment:
hmm, no reason to change the name, I will change it back to Identities at the next PR